examples/speech_to_text/readme.md

```diff
@@ -40,3 +40,23 @@ The results of the evaluation should be as following. The detailed results can b
   BLEU     LAAL       AL     AP       DAL       ATD
  100.0  822.018  822.018  0.581  1061.271  2028.555
 ```
+
+### Example Streaming ASR: Whipser Wait-K model
```

Comment:
Since this agent supports S2T also; should this say streaming ASR/S2T model?

Reply:
Sounds good! As @ibanesh mentioned, whisper only do x->en translation. I also added the option to the agent, but haven't add data for test.